Parent/Parent/ParentAppDelegate.swift

```diff
@@ -275,7 +275,14 @@ extension ParentAppDelegate: NativeLoginManagerDelegate {
         guard let token = client.authSession.token else {
             return
         }
-        let entry = KeychainEntry(accessToken: token, baseURL: client.authSession.baseURL, expiresAt: nil, locale: "en", masquerader: nil, refreshToken: nil, userAvatarURL: client.authSession.user.avatarURL, userID: client.authSession.user.id, userName: client.authSession.user.name)
+
+        let masquerader = client.originalBaseURL != nil
```

Comment:
I think `originalBaseURL` might be nil if you masquerade within your same org.

Reply:
Oh good to know. I'll change this up.